vft505qa.qmb.txt

```diff
@@ -0,0 +1,21 @@
+﻿Albanian Lang (not complete)
+
+Signed-off-by: gertpere <gert.pere@gmail.com>
```

Comment:
I guess you tried to sign off your commit message. Somehow there is now this additional file, which we have to delete again.